using-timescaledb/compression.md

```diff
@@ -0,0 +1,412 @@
+following command:
+
+``` sql
+select compress_chunk( '_timescaledb_internal._hyper_1_2_chunk');
+```
+You can see the results of the compression of that given chunk by running the following:
+
+``` sql
+select * from _timescaledb_catalog.compression_chunk_size
+order by chunk_id;
+```
+This will return a result set that will show you the compressed chunks and the 
+stats about those chunks. In the case of a single chunk the results look like this:
+
+We could then proceed to compress all of the chunks in this example that are more 
+than three days old by simply repeating the process for the remaining chunks in 
+the list we generated.
+
+Policy Based Compression:
+
+We have covered the concept of manually going in and identifying what chunks you 
+would like to see compressed, next we will cover automating this process with policy 
+based compression.  
+
+With policy based compression, we can tell TimescaleDB to set a policy that will 
+compress chunks of data when they reach a given age. As we covered in an earlier 
+section, a chunk becomes eligible for compression as soon as it is no longer the 
+“active” chunk, that is it has moved from position zero to position one (in our 
+diagram above). When this happens is determined by when you create the hypertable 
+using a time partition.
+
+For example, to compress chunks older than 60 days on a hypertable named conditions:
+
+``` sql
+select add_compress_chunks_policy('conditions', '60d'::interval);
+```
+
+This will create a policy that will ensure all chunks older than 60 days will be 
+compressed.  Please note this still requires that we set up the hypertable for compression 
+and that you have issued the commands to ensure that TimescaleDB understands how 
+to organize your data. This command simply automates the process of compressing 
+those chunks that cross the 60 day age threshold.
+
+To confirm that your policy job has been created, you can validate it by using 
+the following command:
+
+``` sql
+select * from _timescaledb_config.bgw_job where job_type like 'compress%';
+```
+
+Notice the system will look for chunks that cross the 60 day threshold every 15 
+minutes and if the job for some reason fails it will be put in the retry queue 
+and be retried in an hour.
+---
+
+### Decompressing Chunks [](decompress)
+
+Next we want to walk through what needs to happen in the event that you need to 
+backfill or update data that lives in a compressed chunk. 
+
+TimescaleDB does not support inserts orupdates into a compressed chunk. To insert 
+or update data, we must first decompress the target chunks. But before we can do 
+that, we need to turn off our compression policy, otherwisethat policy will attempt 
+to re-compress the chunks that we are currently working on (not the desired result). 
+To accomplish this we will issue the following command:
+
+``` sql
+select remove_compress_chunks_policy('conditions');
```

Comment:
note we probably should be using alter_job_policy(job_id, next_run=>'infinity'); instead. But, maybe better left till later

Reply:
agreed!!!